src/components/FormValidator.js

```diff
@@ -1,23 +1,23 @@
 export default class FormValidator {
   constructor(config, formElement) {
     this._inputSelector = config.inputSelector;
-    this._submitButtonSelector = config.submitButtonSelector;
+    //this._submitButtonSelector = config.submitButtonSelector;
     this._inactiveButtonClass = config.inactiveButtonClass;
     this._inputErrorClass = config.inputErrorClass;
     this._errorClass = config.errorClass;
 
     this._element = formElement;
+
+    this._submitButton = this._element.querySelector(".modal__button");
```

Comment:
`".modal__button"` should be taken from `config` not to hardcode the string

Reply:
Ok! I am currently figuring this one out despite my submission. I may have a snag somewhere along the way when I use that selector with my disable and enable methods.. thank you!